ci-operator/step-registry/osd-ccs/fleet-manager/osd-ccs-fleet-manager-workflow.yaml

```diff
@@ -0,0 +1,11 @@
+workflow:
+  as: osd-ccs-fleet-manager
+  steps:
+    pre:
+    - chain: osd-ccs-fleet-manager-provision
```

Comment:
The workflow is the lifecycle on provisioning/deprovisioning the mc/sc clusters via the fleet manager API, but not provisioning/deprovisioning the fleet manager.